package/MDAnalysis/analysis/pca.py

```diff
@@ -123,32 +129,83 @@ class PCA(AnalysisBase):
     principal components ordering the atom coordinate data by decreasing
     variance will be available for analysis. As an example:
 
-        >>> pca = PCA(universe, select='backbone').run()
-        >>> pca_space =  pca.transform(universe.select_atoms('backbone'), 3)
+    .. code-block:: python
+        pca = PCA(universe, select='backbone').run()
+        pca_space = pca.transform(universe.select_atoms('backbone'), 3)
+
 
     generates the principal components of the backbone of the atomgroup and
     then transforms those atomgroup coordinates by the direction of those
     variances. Please refer to the :ref:`PCA-tutorial` for more detailed
     instructions.
 
+    Parameters
```

Comment:
Seems fine to me, but just checking; @lilyminium @orbeckst is this the new style we are now following? (I didn't follow the whole docstring conversation too closely it seems..)

Reply:
At least we used this style in #3233

Reply:
We are moving towards documenting classes in the class string instead of `__init__`... IIRC. However, right now I am not overly concerned with where the docs are, as long as information is either in class or init doc string with the correct attributes etc I am happy. 

I support @PicoCentauri 's edit here (and move of examples into class doc strings in other PRs) — but I am not going to hold up any other PR if it doesn't do exactly the same thing. If someone wants to raise (or find...?) and issue to decide on class doc then please do. But it can really wait post 2.0.0.